Godeps/Godeps.json

```diff
@@ -2791,6 +2791,58 @@
 			"Rev": "c2ac40f1adf8c42a79badddb2a2acd673cae3bcb"
 		},
 		{
+			"ImportPath": "k8s.io/metrics/pkg/apis/custom_metrics",
```

Comment:
The entanglement is more complex than i thought. Apart from the core client, k8s.io/metrics is [importing]("https://github.com/kubernetes/kubernetes/blob/master/vendor/k8s.io/metrics/pkg/apis/metrics/v1alpha1/types.go#L21") "k8s.io/client-go/pkg/api/v1" as well. It's going to block us from moving external types to another repo in https://github.com/kubernetes/kubernetes/pull/44784.

One expedient solution is moving the entire vendor/k8s.io/metric to thirdparty/forked and manually take care the dependency until we get rid of the staging. @sttts @DirectXMan12 @lavalamp how's that sound?